app/src/main/java/com/example/triptracker/MainActivity.kt

```diff
@@ -72,6 +74,9 @@ class MainActivity : ComponentActivity() {
 
             composable(Route.RECORD) { RecordScreen(context, navigation) }
             composable(Route.PROFILE) { UserProfileOverview(navigation = navigation) }
+            // composable(Route.FRIENDS) { UserProfileFriends(navigation = navigation) }
+            composable(Route.FOLLOWERS) { UserProfileFollowers(navigation = navigation) }
+            composable(Route.FOLLOWING) { UserProfileFollowing(navigation = navigation) }
```

Comment:
Will we use Friends as a route? Or is Followers / Following model sufficient ? If this is the case, could you please remove the commented line?